test/helpers/shouldFail.js

```diff
@@ -0,0 +1,31 @@
+const should = require('chai')
+  .should();
+
+async function shouldFailWithMessage (promise, message) {
+  try {
+    await promise;
+  } catch (error) {
+    error.message.should.include(message, 'Wrong failure type');
+    return;
+  }
+
+  should.fail(`Expected '${message} failure not received`);
+}
+
+function reverting (promise) {
+  return shouldFailWithMessage(promise, 'revert');
+}
+
+async function throwing (promise) {
+  return shouldFailWithMessage(promise, 'invalid opcode');
+}
+
+async function outOfGas (promise) {
```

Comment:
We're not really using this one, so why have it at all? Same for `throwing`.

Reply:
We'll want to have them once we publish the helpers and remove them from this repo (part of https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1204), I'd just leave them be instead of removing them and later bringing them back.